asreview/utils.py

```diff
@@ -20,6 +20,8 @@
 import numpy as np
 import pkg_resources
 
+from asreview.webapp.authentication.models import SingleUser
```

Comment:
Remove this import, it isn't used and utils should not be importing from asreview.

Reply:
Done